src/python/turicreate/toolkits/image_analysis/image_analysis.py

```diff
@@ -62,7 +62,9 @@ def load_images(
     ...                                                       recursive=True)
     """
     from ... import extensions as _extensions
+    from ...util import _make_internal_url
```

Comment:
mixing relative and absolute is not a good idea in our project. We'd better choose one style and be coherent with it.

Reply:
I haven't heard of that being a problem. What's your concern?

Reply:
> Relative imports for intra-package imports are highly discouraged. Always use the absolute package path for all imports. Even now that PEP 328 [7] is fully implemented in Python 2.5, its style of explicit relative imports is actively discouraged; absolute imports are more portable and usually more readable.

Reply:
or you can find it SO,
https://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module